tidy3d/components/data/monitor_data.py

```diff
@@ -1972,6 +1974,32 @@ class FluxTimeData(MonitorData):
     )
 
 
+class DirectivityData(MonitorData):
+    """Data associated with a :class:`.DirectivityMonitor`.
+    Example
+    -------
+    >>> from tidy3d import DirectivityDataArray
+    >>> f = np.linspace(1e14, 2e14, 10)
+    >>> r = np.atleast_1d(5)
+    >>> theta = np.linspace(0, np.pi, 10)
+    >>> phi = np.linspace(0, 2*np.pi, 20)
+    >>> coords = dict(r=r, theta=theta, phi=phi, f=f)
+    >>> scalar_field = DirectivityDataArray(farfield_values, coords=coords)
+    >>> monitor = DirectivityMonitor(center=(1,2,3), size=(2,2,2), freqs=f, name='n2f_monitor', phi=phi, theta=theta)
+    >>> data = DirectivityData(monitor=monitor,Directivity=scalar_field)
+    """
+
+    monitor: DirectivityMonitor = pd.Field(
+        ...,
+        title="DirectivityMonitor",
```

Comment:
for variable / class names, put inside of double backslash quotes inside of the Fields.

However, here I think we want this to be read as plain text, so maybe
```
title="Directivity Monitor",
description="Monitor describing the angle-based projection grid on which to measure directivity data."
```

Reply:
Corrected. I used lowercase for 'monitor' to maintain consistency with other monitors.
`title="Directivity monitor",`